### ISSUE TITLE
KMS Encryption rule for aws_elasticache_replication_group

### ISSUE THREAD

**Issue by kmcquade** [bridgecrewio/checkov#1423]
aws_elasticache_replication_group supports KMS key Encryption https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elasticache_replication_group#kms_key_id

There should be a rule to check for Elasticache encryption.

**Compliant code example**

```hcl
resource "aws_elasticache_replication_group" "example" {
  # Removed irrelevant parameters; the ones we care about are below
  at_rest_encryption_enabled = true
  kms_key_id = "${module.encryption_module.kms_key_arn}"
}
```

**Comment by JamesWoolfenden** [comment]
There exists already:
CKV_AWS_29: "Ensure all data stored in the Elasticache Replication Group is securely encrypted at rest"
CKV_AWS_30: "Ensure all data stored in the Elasticache Replication Group is securely encrypted at transit"

**Comment by gruebel** [comment]
thanks to @JamesWoolfenden this is now part of `checkov` and has the check ID `CKV_AWS_191`, which is available since version `2.0.580`.
